modules/manage/pages/security/authorization/rbac.adoc

```diff
@@ -0,0 +1,247 @@
+==============
+NAME  TYPE
+foo   User
+? Confirm deletion of role "red"?  This action will remove all associated ACLs and unassign role members Yes
+Successfully deleted role "red"
+----
+
+=== Assign a role
+
+Administrators may assign a role to any principal created in their Redpanda instance. Principals are referred to using the format: `Type:Name`. Redpanda currently supports only the `User` type. If you omit the type, Redpanda assumes the `User` type by default. With this command you may assign the role to multiple principals at the same time by using a comma separator between each principal.
+
+Using `rpk`, administrators assign a role to a principal by issuing the command:
+
+[,bash]
+----
+rpk security role assign [ROLE_NAME] --principal [PRINCIPALS...]
+----
+
+An example of assigning a role:
+[,bash]
+----
+$ rpk security role assign red --principal foo,bar
+Successfully assigned role "red" to
+NAME  PRINCIPAL-TYPE
+foo   User
+bar   User
+----
+
+=== Unassign a role
+
+Administrators may remove a role assignment from a principal without deleting the role. Principals are referred to using the format: `Type:Name`. Redpanda currently supports only the `User` type. If you omit the type, Redpanda assumes the `User` type by default. With this command you may remove the role from multiple principals at the same time by using a comma separator between each principal.
+
+Using `rpk`, administrators remove a role assignment from a principal by issuing the command:
+
+[,bash]
+----
+rpk security role unassign [ROLE_NAME] --principal [PRINCIPALS...]
+----
+
+An example of unassigning a role:
+[,bash]
+----
+$ rpk security role unassign red --principal bar
+Successfully unassigned role "red" from
+NAME  PRINCIPAL-TYPE
+bar   User
+----
+
+=== Edit role permissions
+
+Using `rpk`, administrators can modify an existing role by adding additional ACLs to it using the command:
+
+[,bash]
+----
+rpk acl create ... --allow-role [ROLE_NAME]
+rpk acl create ... --deny-role [ROLE_NAME]
+----
+
+Administrators may also use `rpk` to remove ACLs from a role by using the command:
+
+[,bash]
+----
+rpk acl delete ... --allow-role [ROLE_NAME]
+rpk acl delete ... --deny-role [ROLE_NAME]
+----
+
+When using the `rpk acl delete` command, Redpanda deletes all ACLs matching the parameters supplied. Be careful to closely match the exact ACL you wish to delete when using this command. If you supply only the `--allow-role` parameter, for example, Redpanda will delete every ACL granting that role authorization to a resource.
+
+To list all the ACLs associated with a role, administrators may use the command:
+
+[,bash]
+----
+rpk acl list --allow-role [ROLE_NAME] --deny-role [ROLE_name]
+----
+
+See also:
+
+* xref:security/authorization/acl.adoc[Access Control Lists] for more information on defining and using ACLs.
+* xref:reference:rpk/rpk-acl/rpk-acl-create.adoc[]
+* xref:reference:rpk/rpk-acl/rpk-acl-delete.adoc[]
+* xref:reference:rpk/rpk-acl/rpk-acl-list.adoc[]
```

Comment:
Should this section be at the end? or is there a reason why is above `List` and `Describe`?

Reply:
These are references to the ACL modification commands within `rpk`, discrete from the role commands. It's part of editing/managing the ACLs on your Roles.